core/compliance/dsar_service.py

```
@@ -16,6 +17,9 @@
 
 logger = logging.getLogger(__name__)
 
+if TYPE_CHECKING:
+    from app.models.fitchef_support_outcomes import FitChefSupportOutcomeEvent
```

Flake8 (7.3.0)

[error] 'app.models.fitchef_support_outcomes.FitChefSupportOutcomeEvent' imported but unused

(F401)